docs/networking/harvester-network.md

```diff
@@ -114,3 +114,62 @@ To create a new untagged network, go to the **Networks > VM Networks** page and
+
+![](/img/kubeovn-harvester-topology.png)
+
+### Create an Overlay Network
+1. Go to **Networks > Virtual Machine Networks**, and then click **Create**.
+
+2. On the **Virtual Machine Network:Create** screen, specify a name for the network.
+
+3. On the **Basics** tab, select `OverlayNetwork` as the network type.
+
+  Specifying a cluster network is not required because the overlay network is only enabled on `mgmt` (the built-in management network).
+
+4. Click **Create**.
+
+
+### How to use overlay network
+To create a new overlay network, go to the **Networks > VM Networks** page and click the **Create** button. You have to specify the name, select the type `OverlayNetwork`. You don't need to specify the cluster network since the overlay network is only enabled on the default management network.
+
+The overlay network functions as the provider of the subnet that is created in the virtual private cloud. Because of this, each subnet must be mapped to only one overlay network, and each overlay network can be used by only one subnet. This one-to-one relationship ensures that routing behavior is clear and predictable, subnets are isolated, and routing conflicts and traffic leakage are avoided.
+
+
+
+![](/img/create_vm_networks.png)
+
+![](/img/create_vpc.png)
+
+![](/img/create_subnet.png)
+
+### Limitations
+The overlay network implementation in Harvester v1.6 has the following limitations:
+- Overlay networks that are backed by Kube-OVN can only be created on `mgmt` (the built-in management network).
+- The `enableDHCP` and `dhcpV4Options` is not set by default, no default route exists on the virtual machine attached to a Kube-OVN overlay subnet, you can either manually add the subnet’s gateway IP as the virtual machine's default route or use the `managedTap` binding with the following instructions. Attempts to access external destinations fail until the default route configured correctly on the guest operating system.
+Edit attached subnet as yaml and ensure the field `.spec.enableDHCP` is set to `true`
+Edit VM as yaml and modify the interface definition, use binding in network interface
+    ```
+          interfaces:
+          - binding:
+              name: managedtap
+            model: virtio
+            name: default
+    ```
+
+- Underlay networking is still unavailable. Consequently, you cannot directly map a subnet to a physical network, and external hosts cannot reach virtual machines that live on an overlay subnet.
+- The `natOutgoing` field is set to `false` by default in all subnets whether they are created in the default VPC or in a user-defined VPC. If you do not change the value to `true`, virtual machines on the subnet are unable to reach the internet even when the gateway is correctly configured.
+- Static IP defined in cloud-init is ignored for Overlay NICs; the interface will always receive an auto-assigned address from the Subnet’s CIDR.
```

Comment:
This could be refined with more accuracy. The user can still assign whatever IP address they want to the VM's NIC. It's just that if the manually configured IP address does not match what Kube-OVN has allocated for that specific NIC, no connection can be established.

So, in theory, it's possible that the user provided a cloud config to set an IP address that exactly matches what Kube-OVN has allocated for that VM (given that the VM does not come up with a running DHCP client).